com.unity.robotics.urdf-importer/Runtime/UrdfComponents/UrdfJoints/UrdfJointPrismatic.cs

```diff
@@ -74,7 +73,7 @@ public override float GetPosition()
         public override float GetVelocity()
         {
 #if UNITY_2020_1_OR_NEWER
-            return unityJoint.velocity[xAxis];
+            return unityJoint.jointVelocity[xAxis];
```

Comment:
curious is velocity and jointVelocity the same concept?

Reply:
I double-checked it and they are not the same thing. The `velocity` is the linear velocity in the world space, and the `jointVelocity` is the velocity in the reduced space. The `GetPosition()` and `GetEffort()` methods are both in the reduced space, so I imagined that the `GetVelocity()` should be in the reduced space as well.